project-base/storefront/components/Layout/Header/MobileMenu/MobileMenuListItem.tsx

```diff
@@ -0,0 +1,32 @@
+import { MenuItem } from './MobileMenu';
+import { ExtendedNextLink } from 'components/Basic/ExtendedNextLink/ExtendedNextLink';
+import { ArrowIcon } from 'components/Basic/Icon/IconsSvg';
+
+type DropdownMenuListProps = {
+    navigationItem: MenuItem;
+    onExpand: () => void;
+    onNavigate: () => void;
+};
+
+export const DropdownMenuListItem: FC<DropdownMenuListProps> = ({ navigationItem, onExpand, onNavigate }) => {
+    const isWithChildren = !!navigationItem.children?.length;
```

Comment:
Again, you can resolve, as I do not see a reason in setting some strict rules, but do not tell me that `hasChildren` would not be nicer 😆

Reply:
I wouldn't discuss this kind of topics anymore. I think we had already enough discussions about that. I will not convince you about my opinion, but yes, I see it as better naming :) We've agreed to not use only one form so I want to keep it.